tensorrt_llm/_torch/pyexecutor/resource_manager/kv_cache_manager.py

```
@@ -0,0 +1,954 @@
+                    dummy_mrope_position_deltas = torch.zeros(1, dtype=torch.int32).unsqueeze(0)
+                    req.py_multimodal_data["mrope_config"]["mrope_position_deltas"] = (
+                        dummy_mrope_position_deltas
+                    )
+            requests.append(req)
+        return requests
+
+    def update_resources(
+        self,
+        scheduled_batch: ScheduledRequests,
+        attn_metadata: "AttentionMetadata" = None,
+        kv_cache_dtype_byte_size: float = None,
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)